package.json

```diff
@@ -56,7 +59,7 @@
   "devDependencies": {
     "chai-as-promised": "^7.1.1",
     "codecov": "^3.0.2",
-    "nyc": "^12.0.2",
+    "nyc": "git://github.com/istanbuljs/nyc#master",
```

Comment:
Should we change back to latest version, since you already fix the problem without the need of the master branch?